src/test/java/com/netcracker/study/general/GameFieldTest.java

```diff
@@ -0,0 +1,23 @@
+package com.netcracker.study.general;
+
+import com.netcracker.study.objects.borders.Wall;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+public class GameFieldTest {
+
+    @Test
+    public void ConstructFieldTest() throws FileNotFoundException {
+        String testMapPath = "src/test/resources/TestMap.txt";
+        Wall testWall = new Wall(1);
+        Wall testPassage = new Wall(0);
+
+        GameField gameField = new GameField(testMapPath);
+
+        Assertions.assertThrows(IllegalArgumentException.class, () -> {gameField.getBorderReference(10, 20, Direction.RIGHT);});
+        Assertions.assertEquals(testWall, gameField.getBorderReference(0, 1, Direction.BOTTOM));
+        Assertions.assertEquals(testPassage, gameField.getBorderReference(1, 1, Direction.BOTTOM));
```

Comment:
there is a strict rule that one test must have only one assert. If you have multiple asserts then either you should have multiple tests or these asserts are useless

Reply:
separated tests